### ISSUE TITLE
[BUG] Select & gradient tools' selection boxes leave ghost trail

### ISSUE THREAD

**Issue by Pythagoras-314** [aseprite/aseprite#4802]
When you wiggle around the selection and gradient tools (and likely other tools) in aseprite, it creates a dark ghost trail that is eliminated when the select tool is left-clicked with. It is visually distracting and was not mentioned in the changelog, leaving me to believe that it is unintended behavior.

![image](https://github.com/user-attachments/assets/74c4ee9d-2dda-4d84-9a87-960db46b035d)
![image](https://github.com/user-attachments/assets/279f85a8-326e-4584-ab42-2f8e24dff802)

* Aseprite version: v1.3.11-beta1-arm64
* System: MacOS Sequoia 16.0

**Comment by Gasparoken** [comment]
Thanks for this report. I was able to reproduce it. It's something related to color management and rendering of the cursor preview. The effect will stop if you change the `Window Color Profile` to `Use sRGB Color Profile`.
This was verified to only happen in the beta version, whereas 1.3.10.1 does not.

This is not the solution, it's just a workaround to resolve the cursor shadows until the root issue has been fixed.
